<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>USignUp</class>
 <widget class="QMainWindow" name="USignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/bG no arrows.png);
color:rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label
{
background:rgb(134,207,234);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;NAME:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_2
{
background:rgb(134,207,234);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;EMAIL:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_7
{
background:rgb(134,207,234);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;ADDRESS:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_8
{
background:rgb(134,207,234);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;PASSWORD:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_3
{
background:rgb(134,207,234);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;PHONE:&lt;/span&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;(include +92)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_6
{
background:rgb(134,207,234);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;CNIC:&lt;/span&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;without dashes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_5
{
border-radius:10px;
background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_0">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_0
{
border-radius:10px;
background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>300</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_2
{
border-radius:10px;
background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_1
{
border-radius:10px;
background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_4
{
border-radius:10px;
background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>350</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_3
{
border-radius:10px;
background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>530</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Back
{
border-radius:10px;
font:bold;
border:3px solid rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SUBMIT">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#SUBMIT
{
border-radius:10px;
font:bold;
border:3px solid rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_4
{
background:rgb(134,207,234);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#ffffff;&quot;&gt;SIGNUP&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#ffffff;&quot;&gt;FOR&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#ffffff;&quot;&gt;USERS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/background/hmu logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
